packages/react/.storybook/styles.scss

```diff
@@ -5,8 +5,9 @@ h1,
 h2,
 h3,
 h4,
-h5 {
-  font-family: "Overpass", sans-serif !important;
+h5,
+h6 {
```

Comment:
Are these declarations even necessary since they are being defined in monorepo?

Reply:
Unfortunately yea since Storybook has its own font styles on pages that you have to override.